app/scenes/Invite.js

```diff
@@ -67,7 +68,8 @@ class Invite extends React.Component<Props> {
     this.invites.push({ email: '', name: '' });
   };
 
-  handleRemove = (index: number) => {
+  handleRemove = (ev: SyntheticEvent<>, index: number) => {
+    ev.preventDefault();
```

Comment:
This fixes an existing bug where clicking the "X" next to an invite would close the dialog.